modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -1091,7 +1060,10 @@ public int compare(ObjectEntry objectEntry1, ObjectEntry objectEntry2) {
 			int index2 = _ids.indexOf(id2);
 
 			if ((index1 == -1) && (index2 == -1)) {
-				return Long.compare(id1, id2);
+				return objectEntry1.getDateCreated(
+				).compareTo(
+					objectEntry2.getDateCreated()
+				);
```

Comment:
Nice and simple in the end :+1:

Reply:
Yes, I had a much more complex impl which turned out unneeded